menpo/io/test/io_export_test.py

```diff
@@ -219,6 +219,8 @@ def test_export_landmark_ljson_nan_values(mock_open, exists):
     # wrote null values
     first_null = mock_open.mock_calls[97][1][0][1:].strip()
     second_null = mock_open.mock_calls[98][1][0][1:].strip()
+    print(first_null)
```

Comment:
what do you mean, how can hardcoded indices into a random string be a bad test 😛